package_maximizer/adapters/__init__.py

```
@@ -341,3 +343,166 @@ def _parse_pacman_deps(self, raw: str) -> list[str]:
 
         return deps
+
+
+class NpmMetadataAdapter:
+    """Парсер метаданных npm (package-lock.json v2/v3)."""
+
+    def fetch(self, package_name: str) -> PackageMetadata | None:
+        """Получить метаданные через npm view."""
+        try:
+            result = subprocess.run(
+                ["npm", "view", package_name, "json"],
```

Ruff (0.16.3)

[error] Starting a process with a partial executable path

(S607)